pkgs/by-name/sw/sway-scratch/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sway-scratch";
+  version = "0.2.1";
+  src = fetchFromGitHub {
+    owner = "aokellermann";
+    repo = "sway-scratch";
+    rev = "v${version}";
+    hash = "sha256-wfwCg51Pfj4IGqOYK2Y/4If7PzZv9JHXewVWQDgbPko=";
```

Hash needs to follow revision.
`$ nix flake prefetch --json github:aokellermann/sway-scratch/v0.2.1`
```suggestion
    hash = "sha256-1N/33XtkEWamgQYNDyZgSSaaGD+2HtbseEpQgrAz3CU=";
```
Make sure to do the same for cargoHash.